core/src/main/java/hudson/util/MultipartFormDataParser.java

```diff
@@ -71,4 +71,23 @@ public void cleanUp() {
         for (FileItem item : byName.values())
             item.delete();
     }
+
+    public static boolean isMultipart(String contentType) {
```

Comment:
I think this should also be `isMultiPartForm` as there are other types of multipart content types.